core/src/main/java/eu/fasten/core/data/callableindex/RocksDao.java

```diff
@@ -174,7 +174,9 @@ public void saveToRocksDb(final ExtendedGidGraph extendedGidGraph) throws IOExce
         // Gather data by source and store it in lists of GraphMetadata.ReceiverRecord.
         edgesInfo.forEach((pair, record) -> map.compute(pair.getFirst().longValue(), (k, list) -> {
             if (list == null) list = new ArrayList<>();
-            list.add(new ReceiverRecord(record.getLine(), transformCallType(record.getCallType()), gidToUriMap.get(pair.getFirst()), Arrays.stream(record.getReceiverTypeIds()).map(typeMap::get).collect(Collectors.toList())));
+            list.add(new ReceiverRecord(record.getLine(), transformCallType(record.getCallType())
+                , gidToUriMap.get(pair.getSecond()),
```

Comment:
So this is the fix itself? i.e., using `pair.getSecond()` instead of `pair.getFirst()`.

Reply:
Yes, that's right